prometheus_api_client/prometheus_connect.py

```diff
@@ -64,6 +64,19 @@ def __init__(
         self._session = requests.Session()
         self._session.mount(self.url, HTTPAdapter(max_retries=retry))
 
+    def check_prometheus_connection(self, params: dict = None) -> bool:
+        """Check database metrics (Prometheus) connection."""
+        response = self._session.get(
+            "{0}/".format(self.url),
+            verify=self.ssl_verification,
+            headers=self.headers,
+            params=params,
+        )
+        if not response.ok:
```

Comment:
I think `response.ok` is a bool and you can just directly return it.

Reply:
Updated thanks!